<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProperties</class>
 <widget class="QDialog" name="DialogProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>365</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Module's properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnApply">
     <property name="toolTip">
      <string>Apply properties changes.</string>
     </property>
     <property name="text">
      <string>Apply changes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="AreaProperties">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>465</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="Properties"/>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="toolTip">
      <string>Discart properties changes.</string>
     </property>
     <property name="text">
      <string>Cancel changes</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="btnClose">
     <property name="toolTip">
      <string>Close dialog without apllying any unapplied changes.</string>
     </property>
     <property name="text">
      <string>Close dialog</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
